fio.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: fio
+  version: "3.37"
+  epoch: 0
+  description: Flexible I/O Tester
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - libaio-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/axboe/fio
+      tag: fio-${{package.version}}
+      expected-commit: 028fcc7db00155fe329497d7501184e8fe49342f
+
+  - runs: |
+      ./configure --prefix=/usr
```

Comment:
could use builtin melange pipelines for consistency?

```yaml
- uses: autoconf/configure
- uses: autoconf/make
- uses: autoconf/make-install
```

Reply:
I tried that initially, but it's actually not an autoconf configure script, it's a hand-crafted one that doesn't support some of the configuration options used by that pipeline: https://github.com/axboe/fio/blob/3ed8eea0ee9b5d7de603e9b128e6c05941b99938/configure#L3-L4

It seems like I could still use `autoconf/make` and `autoconf/make-install` but I'm not sure if that's a good pattern to follow given it's not actually autoconf...